spec/requests/vendor_api/get_multiple_applications_spec.rb

```diff
@@ -3,66 +3,51 @@
 RSpec.describe 'Vendor API - GET /api/v1/applications', type: :request do
   include VendorApiSpecHelpers
 
-  it 'returns applications scoped by `provider`' do
-    ucl         = create(:provider, code: 'UCL')
+  it 'returns applications scoped by the authenticating provider' do
     strathclyde = create(:provider, code: 'STR')
```

Comment:
There's a bit of dissonance giving the provider variable a name (and giving the provider itself a `code`) now that there's only one of them. Maybe leave the `provider` argument off the second call to `create_list`? (Which could probably just be a `create`, come to think of it)

Reply:
Done